sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_response_handlers.py

```diff
@@ -46,7 +45,6 @@ def process_batch_error(error):
         raise_error = ClientAuthenticationError
     raise raise_error(response=error.response, error_format=CSODataV4Format)
 
-
```

Comment:
Also since we can use error_map now, can we remove the `ClientAuthenticationError` code above?